examples/preact/src/routes/home/index.js

```diff
@@ -0,0 +1,11 @@
+import { h } from 'preact';
+import style from './style.css';
+
+const Home = () => (
+	<div class={style.home}>
+		<h1>Home</h1>
+		<p>This is the Home component.</p>
```

Comment:
Looks like the tab indent is a lot bigger than we use in other projects...usually it's set to 2 spaces.